packages/nextjs/app/_components/ActiveGrants.tsx

```diff
@@ -0,0 +1,54 @@
+import Image from "next/image";
+import { Address } from "~~/components/scaffold-eth";
+import { getAllActiveGrants } from "~~/services/database/grants";
+import { GrantData } from "~~/services/database/schema";
+import { formatDateFromNow } from "~~/utils/grants";
+
+const ActiveGrantRow = ({ title, askAmount, builder, timestamp }: GrantData) => {
+  return (
+    <tr className="border-b border-black p-10 text-base">
+      <td className="p-4 pl-4">{title}</td>
+      <td className="p-4 pl-4">{askAmount} ETH</td>
+      <td className="p-4 pl-4">
+        <Address address={builder} />
+      </td>
+      <td className="p-4 pl-4">{formatDateFromNow(timestamp)}</td>
+    </tr>
+  );
+};
+
+export const ActiveGrants = async () => {
+  const activeGrants = await getAllActiveGrants();
+
+  return (
+    <div className="container flex flex-col justify-center max-w-[90%] xl:max-w-7xl mx-auto py-16 gap-4">
+      <div className="self-center lg:self-start w-fit relative pb-8">
```

Comment:
Lol this `pb-8` seems too strong on mobile, I think we could completely remove it and since other sections don't have one